docs/geode-native-docs/introduction/developing-linux.html.md.erb

```diff
@@ -19,48 +19,48 @@ See the License for the specific language governing permissions and
+## <a id="developing-linux__section_EFE9D42B05B44F2F958054224BCC5F76" class="no-quick-link"></a>Step 2. Compile C++ Clients and Dynamically Link Them to the Geode Library
 
-On Linux, the `g++` compiler is supported. To build and link a C++ client to GemFire on Linux, the compilation command line must include the arguments listed in the following table.
+On Linux, the `g++` compiler is supported. To build and link a C++ client to Geode on Linux, the compilation command line must include the arguments listed in the following table.
 
 <a id="developing-linux__compiler-arguments-linux"></a>
 
 | Argument           | Explanation                                              |
 |--------------------|----------------------------------------------------------|
 | `-D_REENTRANT`     | Required to compile Linux programs in a thread-safe way. |
 | `-m32` or `-m64`   | Enables 32-bit or 64-bit compilation.                    |
-| `-I$GFCPP/include` | Specifies the native client `include` directory.         |
+| `-I$GEODE/include` | Specifies the client `include` directory.         |
 
 
-The following table lists the linker switches that must be present on the command line when dynamically linking to the GemFire library.
+The following table lists the linker switches that must be present on the command line when dynamically linking to the Geode library.
 
 <a id="developing-linux__table_216E9E87FBB44E39BE26388317F3ACE4"></a>
 
 | Argument            | Explanation                                                                                        |
 |---------------------|----------------------------------------------------------------------------------------------------|
-| `-rpath $GFCPP/lib` | Tells the linker to look in `$GFCPP/lib` for libraries on which the native client library depends. |
-| `-L$GFCPP/lib`      | Tells the linker where to find the named libraries.                                                |
+| `-rpath $GEODE/lib` | Tells the linker to look in `$GEODE/lib` for libraries on which the client library depends. |
+| `-L$GEODE/lib`      | Tells the linker where to find the named libraries.                                                |
 | `-o durableclient`  | Tells the linker to output an object file named 'durableclient'.                                   |
-| `-lgfcppcache`      | Links the native client C++ cache library to the compiled executable.                              |
+| `-lgfcppcache`      | Links the client C++ cache library to the compiled executable.                              |
 
 
-The following examples compile and link the `$GFCPP/SampleCode/quickstart/cpp/DurableClient.cpp` client to the `durableclient` output file.
+The following examples compile and link the `$GEODE/SampleCode/quickstart/cpp/DurableClient.cpp` client to the `durableclient` output file.
 
 **Compiling and Dynamically Linking on Linux for 32-bit**
```

Comment:
It would be better to have the examples be for 64-bit, in most cases any new development would be on a 64-bit machine/OS